src/views/viewer/DiscoverView.tsx

```diff
@@ -0,0 +1,234 @@
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/education.webp',
+  },
+  'travel-and-events': {
+    id: '6',
+    title: 'Travel & Events',
+    icon: <SvgVideoCategoriesTravelAndEvents />,
+    color: '#5A7AEE',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/travel-and-events.webp',
+  },
+  'sports': {
+    id: '5',
+    title: 'Sports',
+    icon: <SvgVideoCategoriesSports />,
+    color: '#41EE5A',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/sports.webp',
+  },
+  'gaming': {
+    id: '7',
+    title: 'Gaming',
+    icon: <SvgVideoCategoriesGaming />,
+    color: '#9455E8',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/gaming.webp',
+  },
+  'comedy': {
+    id: '9',
+    title: 'Comedy',
+    icon: <SvgVideoCategoriesComedy />,
+    color: '#4FE1F2',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/comedy.webp',
+  },
+  'autos-and-vehicles': {
+    id: '2',
+    title: 'Autos & Vehicles',
+    icon: <SvgVideoCategoriesAutosAndVehicles />,
+    color: '#6E5FEC',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/autos-and-vehicles.webp',
+  },
+  'howto-and-style': {
+    id: '12',
+    title: 'Howto & Style',
+    icon: <SvgVideoCategoriesHowtoAndStyle />,
+    color: '#E57827',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/howto-and-style.webp',
+  },
+  'music': {
+    id: '3',
+    title: 'Music',
+    icon: <SvgVideoCategoriesMusic />,
+    color: '#6EEC3A',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/music.webp',
+  },
+  'nonprofits-and-activism': {
+    id: '15',
+    title: 'Nonprofits & Activism',
+    icon: <SvgVideoCategoriesNonprofitsAndActivism />,
+    color: '#E141D6',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/nonprofit-and-activism.webp',
+  },
+  'news-and-politics': {
+    id: '11',
+    title: 'News & Politics',
+    icon: <SvgVideoCategoriesNewsAndPolitics />,
+    color: '#48F0B3',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/news-and-politics.webp',
+  },
+} as const
+const featuredVideoCategories: Array<{ videoUrl: string; videoTitle: string } & VideoCategoryData> = [
+  {
+    ...videoCategories['travel-and-events'],
+    videoUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/featured-videos/5778.mp4',
+    videoTitle: 'Wreck Diving. Philippines, Boracay.',
+  },
+  {
+    ...videoCategories['education'],
+    videoUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/featured-videos/3941.mp4',
+    videoTitle: 'ФИЛОСОФИЯ СТОИЦИЗМА',
+  },
+  {
+    ...videoCategories['film-and-animation'],
+    videoUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/featured-videos/5984.mp4',
+    videoTitle: 'taxi driver edit',
+  },
+]
```

Comment:
Let's move this out of this file. Not sure where, so maybe just an adjacent `config.ts` (although you will need `DiscoverView` directory)